sentry_sdk/integrations/pymongo.py

```diff
@@ -163,6 +163,7 @@ def started(self, event):
 
             for tag, value in tags.items():
                 span.set_tag(tag, value)
```

Comment:
We should include a comment to indicate that the `set_tag` call is included for backwards-compatibility. Something like the following could do:

```suggestion
                # set the tag for backwards-compatibility.
                # TODO: remove the set_tag call in the next major release!
                span.set_tag(tag, value)

```

Reply:
Good point!